rotkehlchen/db/upgrades/v41_v42.py

```diff
@@ -118,15 +118,38 @@ def _add_calendar_table(write_cursor: 'DBCursor') -> None:
     );""")  # noqa: E501
 
 
+@enter_exit_debug_log()
+def _remove_manualcurrent_oracle(write_cursor: 'DBCursor') -> None:
+    """Removes the manualcurrent oracle from the current_price_oracles setting"""
+    write_cursor.execute('SELECT value FROM settings WHERE name="current_price_oracles"')
+    if (data := write_cursor.fetchone()) is None:
+        return  # oracles not configured
+
+    try:
+        oracles: list[str] = json.loads(data[0])
+    except json.JSONDecodeError as e:
+        log.error(f'Failed to read oracles from user db. {e!s}')
+        return
+
+    write_cursor.execute(
+        'INSERT OR REPLACE INTO settings(name, value) VALUES(?, ?)',
+        (
```

Comment:
nitpick, but making it compact
```suggestion
        'INSERT OR REPLACE INTO settings(name, value) VALUES(?, ?)', (
```

Reply:
I used Black for formatting to ensure consistency. Should I adjust this to fit a different style preference?

Reply:
no issues then, just make sure `make lint` passes